asv_bench/benchmarks/series_methods.py

```diff
@@ -71,4 +71,24 @@ def setup(self):
 
     def time_series_nsmallest2(self):
         self.s2.nsmallest(3, take_last=True)
-        self.s2.nsmallest(3, take_last=False)
\ No newline at end of file
+        self.s2.nsmallest(3, take_last=False)
+
+
+class series_dropna_int64(object):
+    goal_time = 0.2
+
+    def setup(self):
+        self.s = Series(np.random.randint(1, 10, 1000000))
+
+    def series_dropna_int64(self):
```

Comment:
Also this `dropna` only has effect for each time `setup` is run, which is not every iteration
And set seed (not sure)?